### PR TITLE
Project documentation 

### PR DESCRIPTION
**Pull request by Abd210**
### Pull Request Overview

This pull request adds/changes/fixes suggested comments.

### Testing Strategy

This pull request was tested by Mahmoud Mirghani Abdelrahman

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by:  Mahmoud Mirghani Abdelrahman <abdul.mirghani@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/abdul.mirghani/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Object Detection and Alert System
+Automated Surveillance and Notification Interface.
+
+:::info
+**Author**: Mahmoud Mirghani Abdelrahman
+
```
Comment (alexandruradovici) [inline docs/project/2024/abdul.mirghani/index.md:6]:
```suggestion
**Author**: Mahmoud Mirghani Abdelrahman \
```

File: docs/project/2024/abdul.mirghani/index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+|[embassy-time](https://embassy.dev/book/dev/time_keeping.html)|Time management library  |Used for time-based operations such as delays |
+|[embassy-rp](https://docs.embassy.dev/embassy-rp/git/rp2040/index.html)| Peripheral access library |Used for initializing and interacting with peripherals |
+|[log](https://docs.embassy.dev/embassy-usb-logger/git/default/index.html)|Logging facade |Used for logging messages |
+|[embassy-usb-logger](https://docs.embassy.dev/embassy-usb-logger/git/default/index.html)|USB logger implementation for embassy  |Used for logging messages over USB  |
+|[gpio](https://docs.embassy.dev/embassy-stm32/git/stm32c011d6/gpio/index.html)|GPIO manipulation |Used for interacting with GPIO pins |
+|[HC-SR04](https://github.com/Benehiko/pico-ultrasonic-rs.git) |HC-SR04 manipulation| 
+Used for controlling the ultrasonic sensor|
+## Links
+
+1. [youtube](https://youtube.com/shorts/o7DMHJKhpws?si=-lUkMuV3EGYiiGDN)
```
Comment (alexandruradovici) [inline docs/project/2024/abdul.mirghani/index.md:121]: Please add a title to the link, not just *youtube*.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/abdul.mirghani/index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+3. Servo Motor: Moves the ultrasonic sensor back and forth in an arc, mimicking radar scanning.
+  
+4. Buzzer: Provides an audio alert when an obstacle is detected.
+  
+5. LCD Screen: Displays real-time feedback on the distance to detected objects and the angle of the servo motor.
+Detail in a few words the hardware used.
+
+### Schematics
+
+Place your KiCAD schematics here.
```
Comment (alexandruradovici) [inline docs/project/2024/abdul.mirghani/index.md:90]:
```suggestion
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — DISMISSED** [review]

**Comment by alexandruradovici** [comment]
Please resolve the conflicts. It seems to me that your project has been already added.

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Comment by Abd210** [comment]
> Please resolve the conflicts. It seems to me that your project has been already added.

is it okay now?

**Review by alexandruradovici — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**